dialects/core/normalizers.cpp

```diff
@@ -18,17 +121,13 @@ constexpr std::array<std::array<u64, 2>, 2> make_truth_table(bit2 sub) {
 }
 
 // clang-format off
-
-// we rely on dependent lookup, so these cannot be overloads, but instead have to be
-// template specializations
+// We rely on dependent lookup, so these cannot be overloads, but instead have to be template specializations.
```

Comment:
I believe this comment implies, that the `rop` and `rcmp` overloads have to be specializations as well.
Though, I am not quite sure, whether this is still needed, since we don't have this in the Thorin base anymore.